api/primary.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/docker/swarm/cluster"
+	"github.com/docker/swarm/pkg/authZ"
```

Comment:
I guess it's fine to have it under `pkg` as this one is created for the purpose of the PR. Having it under Godep would mean having it under a separate repo, but we want to be able to fix things asap without having to update dependencies with Godeps at first (can be considered later on but for now it's best to keep it under `pkg` for now imho).